docker-compose.yaml

```diff
@@ -56,3 +56,11 @@ services:
       - LDAP_ADMIN_PASSWORD=adminpassword
       - LDAP_USERS=alice,bob,foo
       - LDAP_PASSWORDS=password1,password2,password3
+
+  postgres:
```

Comment:
Do we really need an entirely new container for this?

Can we just reuse the existing mysql container for this and the existing rotate root test [TestAccDatabaseSecretBackendConnectionTemplatedUpdateExcludePassword_mysql](https://github.com/hashicorp/terraform-provider-vault/blob/76aec3409c93fc78f4b5d8d2b0d055af454a377b/vault/resource_database_secret_backend_connection_test.go#LL582C6-L582C80)? If we do end up reusing that test can we just assert that the user and password are not changed in the TF state after the rotate root operation?

Reply:
That seems reasonable to use mysql to test this change. I'm fine either way, but leaning towards a new unit test to make it obvious what we're testing for. @vinay-gopalan thoughts? I know you had originally wrote the test for Postgres.

Reply:
I don't want this comment to block progress. So feel free to disregard if you see fit. However, it seems to me that the `TestAccDatabaseSecretBackendConnectionTemplatedUpdateExcludePassword_mysql` test is intended to test this exact scenario but it fails to assert that the password is unchanged in the state after the rotate root operation.

Reply:
But, presumably, TestAccDatabaseSecretBackendConnectionTemplatedUpdateExcludePassword_mysql would have failed that assertion before this change so maybe I am wrong there.